packages/genesys-spark-tokens/snapshot/css/gse-ui-formsAndInputs.css

```diff
@@ -301,14 +308,9 @@
   --gse-ui-contextMenu-button-default: 32px;
   --gse-ui-contextMenu-button-compact: 24px;
   --gse-ui-contextMenu-borderRadius: 4px;
-  --gse-ui-contextMenu-states-default-foregroundColor: #767c8d;
-  --gse-ui-contextMenu-states-hover-foregroundColor: #3742a2;
-  --gse-ui-contextMenu-states-hover-backgroundColor: #dee1fd;
-  --gse-ui-contextMenu-states-active-foregroundColor: #252e78;
-  --gse-ui-contextMenu-states-active-backgroundColor: #ced2fc;
-  --gse-ui-contextMenu-states-disabled-foregroundColor: #767c8d;
   --gse-ui-contextMenu-disabled-opacity: 0.5;
-  --gse-ui-contextMenu-gap: 4px;
```

Comment:
This is used (hardcoded in the tsx file at the moment). Why was it removed?

Reply:
for the --gse-ui-contextMenu-disabled-opacity: 0.5 we are using the opacity from buttons now so it won't be used anymore.
--gse-ui-contextMenu-gap: 4px Thomas has reported that this token is not been used because the value has to be specified in the code for external library.